storage/store.go

```
@@ -1649,6 +1665,25 @@ func (s *Store) updateStoreStatus() {
 	}
 }
 
+// PublishStatus publishes periodically computed status events to the store's
+// events feed. This method itself should be periodically called by some
+// external mechanism.
+func (s *Store) PublishStatus() {
+	// broadcast store descriptor.
+	desc, err := s.Descriptor()
+	if err != nil {
+		log.Error(err)
```

why not return this error?